lucille-parent/pom.xml

```diff
@@ -42,10 +42,10 @@
     <maven-javadoc-plugin.version>3.3.2</maven-javadoc-plugin.version>
     <maven-source-plugin.version>3.3.0</maven-source-plugin.version>
     <maven-gpg-plugin.version>3.0.1</maven-gpg-plugin.version>
-    <jackson.version>2.12.2</jackson.version>
+    <jackson.version>2.13.5</jackson.version>
     <!-- todo if updated to version 9 will need to add solr-solrj-zookeeper dependency -->
-    <lucene.version>9.7.0</lucene.version>
-    <solr.version>9.3.0</solr.version>
+    <lucene.version>9.8.0</lucene.version>
+    <solr.version>9.4.1</solr.version>
```

Comment:
Should we go to Solr 9.5?

I got a successful local build with:

    <lucene.version>9.9.2</lucene.version>
    <solr.version>9.5.0</solr.version>

Reply:
With the last commit I've increased these versions as well as a couple other packages.